index.html

```diff
@@ -14,6 +14,56 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header class="header">
+      
+        <nav class="nav_link">
+            <img src="./img/karma-logo.svg" alt="">
+            <ul>
+                <li><a>Meet Karma</a></li>
+                <li><a>How it Works</a></li>
+                <li><a>Store</a></li>
+                <li><a>Blog</a></li>
+                <li><a>Help</a></li>
+                <li><a>Login</a></li>
+
+            </ul>
+        </nav>
+    </header>
+    <article class="main-container">
```

Comment:
```suggestion
    <main>
    <article class="main-container">
```
It looks like this `article` and `section` are part of the main section of the webpage, so we should wrap them in a `main` element